roofit/roostats/inc/RooStats/ToyMCImportanceSampler.h

```diff
@@ -45,9 +45,9 @@ class ToyMCImportanceSampler: public ToyMCSampler {
 
       virtual ~ToyMCImportanceSampler();
 
-      // overwrite GetSamplingDistributionsSingleWorker(paramPoint) with a version that loops
-      // over nulls and importance densities, but calls the parent
-      // ToyMCSampler::GetSamplingDistributionsSingleWorker(paramPoint).
+      /// overwrite GetSamplingDistributionsSingleWorker(paramPoint) with a version that loops
```

Comment:
/// Overwritten implementation of GetSamplingDistributionsSingleWorker(paramPoint) ....